tests/manage/storageclass/test_new_sc_rbd_replica2_3.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pod import get_fio_rw_iops
+from ocs_ci.framework.testlib import ManageTest, tier1
+
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+class TestCreateNewScWithNeWRbDPool(ManageTest):
+    """
+    Create a new  Storage Class on a new rbd pool with
+    different replica and compression options
+    """
+    @pytest.mark.parametrize(
+        argnames=["replica", "compression"],
+        argvalues=[
+            pytest.param(
+                *[2, 'aggressive']
+            ),
+            pytest.param(
+                *[3, 'aggressive']
+            ),
+            pytest.param(
+                *[2, 'none']
+            ),
+            pytest.param(
+                *[3, 'none']
+            ),
+        ]
+    )
+    def test_new_sc_new_rbd_pool(self, replica, compression,
+                                 storageclass_factory, pvc_factory, pod_factory
+                                 ):
+        """
+        This test function does below,
+        *. Creates Storage Classe with creating new rbd pool
+        *. Creates PVCs using each Storage Class
+        *. Mount each PVC to an app pod
+        *. Run IO on each app pod
+        """
+        interface_type = constants.CEPHBLOCKPOOL
+        sc_obj = storageclass_factory(interface_type=interface_type,
+                                      new_rbd_pool=True, replica=replica, compression=compression
+                                      )
+
+        log.info(f"Creating a PVC using {sc_obj.name}")
```

Comment:
instead of lines 50-67, you can use the functionalities from https://github.com/red-hat-storage/ocs-ci/blob/74cbc9f446cb726a9e6781556a94902513ed1bd8/tests/sanity_helpers.py#L21

Reply:
ack